src/photon/test/photon_tests.c

```diff
@@ -0,0 +1,196 @@
+#include "greatest.h"
+
+#include <assert.h>
+#include <unistd.h>
+#include <poll.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/wait.h>
+
+#include "common.h"
+#include "test/test_common.h"
+#include "event_loop.h"
+#include "io.h"
+#include "utstring.h"
+#include "task.h"
+
+#include "photon.h"
+#include "photon_scheduler.h"
+#include "photon_algorithm.h"
+#include "photon_client.h"
+
+SUITE(photon_tests);
+
+const char *plasma_socket_name_format = "/tmp/plasma_socket_%d";
+const char *photon_socket_name_format = "/tmp/photon_socket_%d";
+
+int64_t timeout_handler(event_loop *loop, int64_t id, void *context) {
+  event_loop_stop(loop);
+  return EVENT_LOOP_TIMER_DONE;
+}
+
+typedef struct {
+  /* A socket to mock the Plasma store. */
```

Comment:
I don't feel that strongly about this case, but for structs we should probably document with `/** */`